control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -178,6 +178,8 @@ message PublishVolumeRequest {
   optional string target_node = 2;
   // share protocol
   optional VolumeShareProtocol share = 3;
+  /// Hosts allowed to access nexus.
+  repeated string frontend_hosts = 5;
```

Comment:
4?

Reply:
Now 4 is publish_context from @Abhinandan-Purkait's PR.